website/docs/cloud-docs/vcs/troubleshooting.mdx

```diff
@@ -106,6 +106,8 @@ Terraform Cloud does not allow you to change the API URL for an existing VCS con
 
 If a VCS OAuth connection breaks, you can reauthorize an existing VCS provider while retaining any VCS connected resources, like workspaces. We recommend only using this feature to fix broken VCS connections. We also recommend reauthorizing using the same VCS account to avoid permission changes to your repositories. 
 
+~> **Important:** Reauthorizing is incompatible for OAuth Clients managed by the [TFE Provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/oauth_client).
```

Comment:
Question - in this case, is there anything else they can do, or are they out of luck? If there is an alternative, we should tell them/ at least give them a path forward when something is broken.

```suggestion
~> **Important:** Reauthorizing is not available when the [TFE Provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/oauth_client) is managing the OAuth Client.
```

Reply:
@laurapacilio Great point!

The TFE-provider/terraform equivalent would be to rotate the `oauth_token` argument for the `tfe_oauth_client` resource.

I'll go ahead and add that in.

Reply:
thank you!